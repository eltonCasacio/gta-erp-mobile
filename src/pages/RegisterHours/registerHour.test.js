import React from 'react';
import renderer from 'react-test-renderer';

import RegisterHour from '.';

describe('<RegisterHour', () => {
  it('should render default layout', () => {
    const tree = renderer.create(<RegisterHour />).toJSON();

    expect(tree).toMatchSnapshot();
  });
});
