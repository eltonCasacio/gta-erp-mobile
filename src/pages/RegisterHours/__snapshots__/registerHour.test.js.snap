// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<RegisterHour should render default layout 1`] = `
<View
  style={
    Array [
      Object {
        "alignItems": "center",
        "flexBasis": 0,
        "flexGrow": 1,
        "flexShrink": 1,
        "justifyContent": "space-between",
        "marginBottom": 15,
      },
    ]
  }
>
  <Text
    style={
      Array [
        Object {
          "alignSelf": "flex-start",
          "color": "#fafafa",
          "fontSize": 22,
        },
      ]
    }
  >
    Fulano
  </Text>
  <View
    style={
      Array [
        Object {
          "width": "100%",
        },
      ]
    }
  >
    <View
      style={
        Array [
          Object {
            "alignItems": "center",
            "backgroundColor": "#fafafa",
            "paddingBottom": 0,
            "paddingLeft": 15,
            "paddingRight": 15,
            "paddingTop": 0,
          },
        ]
      }
    >
      <View
        style={
          Array [
            Object {
              "flexDirection": "row",
              "justifyContent": "space-between",
              "marginBottom": 5,
              "marginLeft": 0,
              "marginRight": 0,
              "marginTop": 5,
              "width": "100%",
            },
          ]
        }
      >
        <Text
          checked={true}
          style={
            Array [
              Object {
                "color": "#707070",
                "fontSize": 16,
                "fontWeight": "bold",
              },
            ]
          }
        >
          ENTRADA:
        </Text>
        <Text
          checked={true}
          style={
            Array [
              Object {
                "color": "#707070",
                "fontSize": 16,
                "fontWeight": "bold",
              },
            ]
          }
        >
          06:00
        </Text>
      </View>
      <View
        style={
          Array [
            Object {
              "flexDirection": "row",
              "justifyContent": "space-between",
              "marginBottom": 5,
              "marginLeft": 0,
              "marginRight": 0,
              "marginTop": 5,
              "width": "100%",
            },
          ]
        }
      >
        <Text
          checked={false}
          style={
            Array [
              Object {
                "color": "#AEAEAE",
                "fontSize": 16,
                "fontWeight": "normal",
              },
            ]
          }
        >
          SAÍDA:
        </Text>
        <Text
          checked={false}
          style={
            Array [
              Object {
                "color": "#AEAEAE",
                "fontSize": 16,
                "fontWeight": "normal",
              },
            ]
          }
        >
          10:00
        </Text>
      </View>
      <View
        style={
          Array [
            Object {
              "flexDirection": "row",
              "justifyContent": "space-between",
              "marginBottom": 5,
              "marginLeft": 0,
              "marginRight": 0,
              "marginTop": 5,
              "width": "100%",
            },
          ]
        }
      >
        <Text
          checked={false}
          style={
            Array [
              Object {
                "color": "#AEAEAE",
                "fontSize": 16,
                "fontWeight": "normal",
              },
            ]
          }
        >
          ENTRADA:
        </Text>
        <Text
          checked={false}
          style={
            Array [
              Object {
                "color": "#AEAEAE",
                "fontSize": 16,
                "fontWeight": "normal",
              },
            ]
          }
        >
          12:00
        </Text>
      </View>
      <View
        style={
          Array [
            Object {
              "flexDirection": "row",
              "justifyContent": "space-between",
              "marginBottom": 5,
              "marginLeft": 0,
              "marginRight": 0,
              "marginTop": 5,
              "width": "100%",
            },
          ]
        }
      >
        <Text
          checked={false}
          style={
            Array [
              Object {
                "color": "#AEAEAE",
                "fontSize": 16,
                "fontWeight": "normal",
              },
            ]
          }
        >
          SAÍDA:
        </Text>
        <Text
          checked={false}
          style={
            Array [
              Object {
                "color": "#AEAEAE",
                "fontSize": 16,
                "fontWeight": "normal",
              },
            ]
          }
        >
          18:00
        </Text>
      </View>
    </View>
  </View>
  <View
    style={
      Array [
        Object {
          "marginBottom": 15,
        },
      ]
    }
  >
    <View
      style={
        Array [
          Object {
            "alignItems": "center",
            "justifyContent": "center",
          },
        ]
      }
    >
      <Text
        style={
          Array [
            Object {
              "color": "#fafafa",
              "fontSize": 40,
            },
          ]
        }
      >
        
      </Text>
      <Text
        style={
          Array [
            Object {
              "color": "#fafafa",
              "fontSize": 40,
            },
          ]
        }
      >
        
      </Text>
    </View>
  </View>
  <View
    accessible={true}
    focusable={true}
    onClick={[Function]}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      Object {
        "alignItems": "center",
        "backgroundColor": "#5762F2",
        "borderRadius": 200,
        "opacity": 1,
        "paddingBottom": 10,
        "paddingLeft": 15,
        "paddingRight": 15,
        "paddingTop": 10,
        "width": 200,
      }
    }
  >
    <Text
      style={
        Array [
          Object {
            "color": "#fafafa",
            "fontSize": 22,
          },
        ]
      }
    >
      REGISTRAR
    </Text>
  </View>
</View>
`;
